tests/integration_test_example_bundles.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import boto3
+import json
+import os
+import requests
+import sys
+import unittest
+import uuid
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.insert(0, pkg_root)
+
+
+class TestExampleBundles(unittest.TestCase):
+
+    DSS_API_URI_BASE = 'https://hca-dss.czi.technology/v1'
+
+    def test_example_bundles(self):
+        for bundle in ExampleBundle.all():
+            self.progress("Processing bundle %s" % bundle.path)
+            self.upload_bundle(bundle)
+            self.check_bundle(bundle)
+
+    def upload_bundle(self, bundle):
+        for file in bundle.files:
+            self.upload_file(file)
```

Comment:
planning ahead, you will want to save the timestamps from the upload operation for the `create_bundle` operation.

while you could set the timestamp during your file upload, that would mean multiple runners of this will collide.